app/views/layouts/_navigation.html.erb

```diff
@@ -33,14 +33,17 @@
   </button>
   <div class="collapse navbar-collapse" id="navbarText">
     <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="/obituaries/community">Community Obituaries</a>
+      </li>
```

Comment:
I rearranged this after working on the README and reflecting on how community obits has become extremely important to us.

Reply:
This is true, its become a much bigger piece than originally anticipated.